### PR TITLE
fix(callout-quote): non-latin typefaces do not support serif

### PR DESCRIPTION
**Pull request by bruno-amorim**
### Related Ticket(s)

[ADCMS-6085](https://jsw.ibm.com/browse/ADCMS-6085)
[ADCMS-5955](https://jsw.ibm.com/browse/ADCMS-5955)

### Description

When using the callout-quote component, the shadowroot styles render the blockquote element in a serif font. (e.g. [mainframe application modernization](https://www.ibm.com/mainframe-application-modernization))

In non-latin fonts, however, there isn't a serif version of those typefaces, leading to the components rendering non-plex characters. (e.g. [MAM in Japanese](https://www.ibm.com/jp-ja/mainframe-application-modernization))

I also took advantage of the opportunity, to write a fix related to the Arabic quotation marks issue here: ADCMS-5955
image

### CODE REVIEW

File: packages/styles/scss/components/callout-quote/_callout-quote.scss [hunk 1]
```diff
@@ -34,6 +35,15 @@
       outline: transparent;
       padding-inline-start: $spacing-07;
     }
+
+    &[lang="ar"],
+    &[lang="cn"],
```
Comment (andy-blum) [inline packages/styles/scss/components/callout-quote/_callout-quote.scss:40]:
```suggestion
    &[lang="zh"],
```

Comment (andy-blum) [reply packages/styles/scss/components/callout-quote/_callout-quote.scss:40]: CN is the country code for China - we want the language code "ZH"

File: packages/styles/scss/components/quote/_quote.scss [hunk 2]
```diff
@@ -27,6 +27,15 @@
       max-inline-size: 80%;
       padding-inline-start: $spacing-05;
     }
+
+    &[lang="ar"],
+    &[lang="cn"],
```
Comment (andy-blum) [inline packages/styles/scss/components/quote/_quote.scss:32]:
```suggestion
    &[lang="zh"],
```

**Review by andy-blum — CHANGES_REQUESTED** [review]

**Review by andy-blum — APPROVED** [review]

**Review by kennylam — APPROVED** [review]

File: packages/web-components/src/components/quote/quote.ts [hunk 3]
```diff
@@ -14,6 +14,7 @@ import StableSelectorMixin from '../../globals/mixins/stable-selector';
 import { QUOTE_TYPES } from './defs';
 import '../horizontal-rule/horizontal-rule';
 import { carbonElement as customElement } from '../../internal/vendor/@carbon/web-components/globals/decorators/carbon-element';
+import { LocaleAPI } from '../../internal/vendor/@carbon/ibmdotcom-services/services/Locale';
```
Comment (m4olivei) [inline packages/web-components/src/components/quote/quote.ts:17]: Found a bug here for downstream adopters. This compiles to:

```
import { LocaleAPI } from "../../internal/vendor/@carbon/ibmdotcom-services/services/Locale.js";
```
Which does not exist and throws errors with at least some bundler configurations. This adopter is using Webpack:

```
ERROR in ./node_modules/@carbon/ibmdotcom-web-components/es/components/quote/quote.js 31:0-96
Module not found: Error: Can't resolve '../../internal/vendor/@carbon/ibmdotcom-services/services/Locale.js' in '/Users/m4olivei/projects/ibm-cms-cloud-carbon-v2/ui.frontend.idl/node_modules/@carbon/ibmdotcom-web-components/es/components/quote'
```

It's also reproducible with a simple vite configuration: https://stackblitz.com/github/m4olivei/carbon-demo/tree/quote-test?file=components.js

Will file an issue and fix.

Comment (m4olivei) [reply packages/web-components/src/components/quote/quote.ts:17]: https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/12034

**Pull request merged**